back-end/API/tests/test_user.py

```
@@ -116,196 +131,191 @@ def test_resetPasswordNormal(client):
-    }
-    response5 = client.post('/user/login', json=data5)
-    assert response5.status_code == 200
-   
-    #Remodification du mot de passe
-    data6 = {
-        "email": "test@gmail.com",
-        "password": "test123"
-    }
-    response6 = client.put('/user/updatePassword', json=data6)
-    assert response6.status_code == 200
+    def test_restPasswordWrongUser(client):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `test_restPasswordWrongUser`

Add return type annotation: `None`

(ANN202)